brave/brave-core
# Do not delete contribution queue items

Resolves https://github.com/brave/brave-browser/issues/7593

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Enable Rewards
- Claim grant
- Visit a verified publisher and wait for it to appear in auto-contribute table
- Wait for auto-contribution to occur
- Ensure that `contribution_queue` item is not deleted and has a valid `completed_at` date

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
